patches/iotjs-tizenrt-release.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/platform/tizenrt/iotjs_main_tizenrt.c b/src/platform/tizenrt/iotjs_main_tizenrt.c
```

Comment:
I'm a bit confused. This change is available on the current master. Why do we need this patch?

Reply:
Related commit: 15fb3135830d139a6c851c5b1d1be65c2bfea601

Reply:
@LaszloLango TizenRT doesn't support `echo $?`, so I should print the test result and get it by parsing result. The result is printed only in debug mode, so I cannot get the test result in release mode. This patch solves this problem by removing the define statement.

Reply:
I understand the purpose of the change, but it is already applied on the latest IoT.js. Anyway the patch is good. Let's land it.